backends/mlu/kernels/cross_entropy_kernel.cc

```diff
@@ -374,7 +375,7 @@ void CrossEntropyWithSoftmaxGradKernel(const Context& dev_ctx,
   VLOG(5) << "[CrossEntropyGrad] rank: " << rank << " use_axis: " << use_axis
           << " axis_dim: " << axis_dim << " n: " << n;
   if ((!soft_label && labels.numel() == n &&
-       (ignore_index == -1 || ignore_index == 255)) ||
+       (ignore_index == -1 || ignore_index == 255 || ignore_index == -100)) ||
```

Comment:
If it's ok to use `ignore_index < 0` condition? if so, also cover these cases by ut.

Reply:
already done